css-three-column-layout/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container-one">
+    <div class="row white center">
+      <h2>Designed for life science. Optimized for SaMD and Digital Health.</h2>
+    </div>
+  </div>
+  <div class="container-two">
+    <div class="row">
+      <div class="column-one-third">
+        <div class="row center positioned-down">
+          <img src="images/merge.png" alt="merging picture">
+        </div>
+        <div class="row white center positioned-down-more">
+        <h3>Streamlined QMS</h3>
+        </div>
+        <div class="row grayish-blue center positioned-down-less">
+        <p>
+          A QMS that includes modules for all stages of the product life cycle from Design Control to CAPA
+        </p>
+        </div>
+        <div class="row background-circle"></div>
+      </div>
+      <div class="column-one-third">
+        <div class="row center positioned-down">
+          <img src="images/cloud-computing.png" alt="cloud computing picture">
+        </div>
+        <div class="row white center positioned-down-more">
+          <h3>Integrates with existing tools</h3>
+        </div>
+        <div class=" row grayish-blue center positioned-down-less">
+          <p>
+            Enzyme QMS integrates with your existing tools such as JIRA, GitHub and Jenkins with Enzyme
+          </p>
+        </div>
+        <div class="row background-circle"></div>
+      </div>
+      <div class="column-one-third">
+        <div class="row center positioned-down">
+          <img src="images/discussion.png" alt="picture of people discussing">
+        </div>
+        <div class="row white center positioned-down-more">
+          <h3>In-depth Onboarding</h3>
+        </div>
+        <div class="row grayish-blue center positioned-down-less">
+          <p>
+            We train your entire team and provide you with support when and as it's needed
+          </p>
+        </div>
+        <div class="row background-circle"></div>
```

Comment:
Instead of having this extra element for the circle that needs to be positioned, you should make the `div` that holds the `img` the circle, which will remove any need for positioning.